### PR TITLE
Map page

### PR DESCRIPTION
**Pull request by heatherbooker**
@dharness  cigar?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,4 @@ node_modules
 npm-debug.log
 react-components.md
 build
-.eslintrc.json
\ No newline at end of file
+.eslintrc.json
```
Comment (dharness) [inline .gitignore]: I think you should push this to version control

File: src/components/Map.jsx [hunk 2]
```diff
@@ -1,26 +1,16 @@
 var React = require('react');
+var world = require('../assets/world.svg');
```
Comment (dharness) [inline src/components/Map.jsx]: I think this is a path to an svg, please name it as such

File: src/components/Map.jsx [hunk 3]
```diff
@@ -1,26 +1,16 @@
...
 
-
-//build components
-var Content = React.createClass({
+var MapImg = React.createClass({
```
Comment (dharness) [inline src/components/Map.jsx:5]: I don't think this needs to be a component

File: src/stylesheets/partials/_general.scss [hunk 4]
```diff
@@ -1,5 +1,5 @@
-.f-nav-logo {
-  font-family: $raleway;
+h1, h2, h3, h4, h5, h6, p, span {
```
Comment (dharness) [inline src/stylesheets/partials/_general.scss]: can we name this file _base.scss instead?

**Comment by dharness** [comment]
This was well done. Just a couple small things

**Comment by heatherbooker** [comment]
@dharness done

**Pull request closed without merging**